CHANGELOG.md

```diff
@@ -81,6 +81,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Selection not being cleared when sending chars through a binding
 - Mouse protocols/encodings not being mutually exclusive within themselves
 - Escape `CSI Ps M` deleting lines above cursor when at the bottom of the viewport
+- Escape `CSI Ps c` honored with a wrong `Ps`
+- Handling of `ESC` with intermediates
```

Comment:
`ESC`s with intermediates have been handled before. We just haven't been filtering `ESC` escapes with invalid intermediates before.

```rust
- Ignore `ESC` escapes with invalid intermediates
```